spec/requests/questions_spec.rb

```diff
@@ -61,4 +102,72 @@
       expect(response.body).not_to include("Delete this")
     end
   end
+
+  describe "#correct_answer" do
```

Comment:
So this test should be in a file to test the model - this should be spec/models/question_spec.rb - perhaps have a look at the model tests in govuk-chat for inspiration?